src/Particular.Analyzers/DateTimeOffsetAnalyzer.cs

```diff
@@ -26,31 +26,31 @@ public override void Initialize(AnalysisContext context)
 
         static void AnalyzeVariableDeclaration(SyntaxNodeAnalysisContext context)
         {
-            if (!(context.Node is VariableDeclarationSyntax declareVariable))
+            if (!(context.Node is VariableDeclarationSyntax declaration))
             {
                 return;
             }
 
-            var type = context.SemanticModel.GetTypeInfo(declareVariable.Type, context.CancellationToken).Type;
+            var type = context.SemanticModel.GetTypeInfo(declaration.Type, context.CancellationToken).Type;
             if (type.ToString() != "System.DateTimeOffset")
             {
                 return;
             }
 
-            var declarator = declareVariable.Variables.First();
+            var declarator = declaration.Variables.FirstOrDefault();
```

Comment:
Should we loop through all the variables here rather than just taking the first?

Reply:
I suppose so. Right now I don't think it would catch this:

```
DateTimeOffset dto1, dto2 = DateTime.Now;
```